src/GraphQL.ApiTests/net6/GraphQL.approved.txt

```diff
@@ -409,6 +417,12 @@ namespace GraphQL
         public static T ToObject<T>(this System.Collections.Generic.IDictionary<string, object?> source)
             where T :  class { }
     }
+    [System.AttributeUsage(System.AttributeTargets.Class | System.AttributeTargets.All)]
```

Comment:
> | System.AttributeTargets.All

Strange. The same with other attributes. I can not reproduce this output on my Windows machine.